src/Our.Umbraco.Ditto/Ditto.cs

```diff
@@ -135,5 +140,26 @@ public static void RegisterDefaultProcessorType<TProcessorAttributeType>()
         {
             TypeDescriptor.AddAttributes(typeof(TObjectType), new TypeConverterAttribute(typeof(TConverterType)));
         }
+
+        /// <summary>
+        /// Registers a global umbraco application context accessor.
+        /// </summary>
+        /// <typeparam name="TUmbracoApplicationContextAccessorType">The type of the accessor.</typeparam>
+        public static void RegisterUmbracoApplicationContextAccessor<TUmbracoApplicationContextAccessorType>()
```

Comment:
Hey @leekelleher, I'd like to have a more concise name however it's an class that acts an accessor for both `UmbracoContext` and `ApplicationContext`. I'm at a loss as to what else to call it but I'm happy to hear any suggestions :)